src/content/docs/logs/logs-everywhere/ruby-telemetry-context.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: 'Ruby telemetry in context'
+metaDescription: For apps monitored by the Ruby agent, learn how to link log data with related data across the rest of the New Relic platform.
+---
+
+[Telemetry in context](https://docswebsitedevelop-doc7761logsintro.gtsb.io/docs/logs/logs-everywhere/telemetry-in-context/) for the Ruby agent connects your logs with all of your telemetry data for your apps, hosts, and other entities. Bringing all of this data together in a single tool helps you quickly:
+
+* Cut through the noise of thousands of logs when troubleshooting time-critical issues, so you automatically see only the most relevant logs.
+* Navigate within multiple types of telemetry data, and have the data correlate back to the original issue.
+* Easily drill down into more detailed information from the same place in the UI.
+* Find the log lines that you need to identify and resolve a problem.
+
+![Telemetry in context in APM UI](./images/ruby-errors-log-details.png "Telemetry in context in APM UI")
+
+<figcaption>
+  **Telemetry in context:** From the left nav in the APM UI, you can select an error and immediately see related logs, log details, host name, important attributes, and more.
+</figcaption>
+
+## Enable and configure [#enable]
+
+You have three options to configure telemetry in context for your application:
+
+<CollapserGroup>
+
+  <Collapser
+    id="1-agent"
+    title="Option 1 (simplest): Let the agent forward your logs."
+  >
+
+  This is the simplest approach, and it's a great choice for developers who may not have the access or interest in setting up a log forwarder, or for accounts that want to see the power of logs in context without a lot of overhead. 
+
+  All you need to do is install an agent version with log forwarding capabilities ([Ruby agent 8.6.0 or higher](https://docs.newrelic.com/docs/release-notes/agent-release-notes/ruby-release-notes/)), and update your configuration to enable forwarding.
+
+  newrelic.yml:
+  ```
+  application_logging.forwarding.enabled: true
```

Comment:
Do we typically show config keys for YAML with dotted notation vs the nested form?

```
application_logging:
  forwarding:
    enabled: true
```

Reply:
Great point, @jasonrclark! We typically do not. I don't know why I chose to write it in that form that time around. I think it's better to use the nested form.